Wikipedia/Networking/Fetchers/WMFENFeaturedTitleFetcher.h

```diff
@@ -0,0 +1,21 @@
+//
+//  WMFFeedItemExtractFetcher.h
+//  Wikipedia
+//
+//  Created by Brian Gerstle on 11/9/15.
+//  Copyright © 2015 Wikimedia Foundation. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@class AnyPromise;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface WMFENFeaturedTitleFetcher : NSObject
```

Comment:
`English` - spell it out, especially if its confusingly jammed up against the prefix.